pallets/stateful-storage/src/weights.rs

```diff
@@ -17,11 +17,11 @@
 // --chain=frequency-bench
 // --heap-pages=4096
 // --wasm-execution=compiled
-// --additional-trie-layers=5
 // --steps=20
 // --repeat=10
 // --output=./scripts/../pallets/stateful-storage/src/weights.rs
 // --template=./scripts/../.maintain/frame-weight-template.hbs
+// --additional-trie-layers=6
```

Comment:
since we passed 1_000_000 I'm increasing this to 6

log<sub>16</sub>(1000000) ~ 5